src/extension.ts

```diff
@@ -374,12 +374,14 @@ function provideHover(document, position, token, syntax): Promise<vscode.Hover>
         } catch (err) {
             // Bad document
             resolve(null);
+            vscode.window.showInformationMessage(`Error: ${err}`);
```

Comment:
I am not sure we want to bug the user with these notifications.  These hovers get attached to all JSON and YAML documents: we are not guaranteed that the document the user is looking at is a k8s resource, nor that it is even valid JSON or YAML at this moment (they may be in the middle of editing it).  It's appropriate not to explain in this case (hence the `resolve(null)`, but we shouldn't put a message in the user's face.

Reply:
yep, deleted.